website/src/components/pageStateSelectors/WasapPageStateSelector.tsx

```diff
@@ -271,23 +291,71 @@ function ResistanceMutationsFilter({
 function UntrackedFilter({
     pageState,
     setPageState,
+    cladeLineageQueryResult: { isPending, isError, error, data: cladeLineages },
```

Comment:
Did you try whether you can simply move the query in here instead of passing the results? Actually `useQuery` should cache the results based on the query key. I think it should work that we don't need to refetch when this component unmounts and is then rendered again.

Reply:
Hm, good point. I don't know from the top of my head. We can leave it as it is :+1: